profile.js

```diff
@@ -1,64 +1,57 @@
-//An attempt to create renderable profile content:
-
 function renderProfile(profile) {
+
+  var $profile = document.createElement('div')
+  var $cover = document.createElement('img')
+  $cover.src = profile.cover
```

Comment:
This isn't working because your `profile` object doesn't have a `cover` property, it's `coverPhoto`.